test/support/test_config.h

```diff
@@ -106,4 +106,12 @@
 
 #define TEST_HAS_NO_INT128
 
+#if defined(_LIBCPP_VERSION) || defined(_MSC_VER)
+#    define TEST_OF_COMPLEX_POW_BROKEN           0

```

Comment:
we have some practice for naming such "broken" macros - please see `onedpl\test\support\test_config.h` - should be included info about compiler and library and "issue title".
For example, _PSTL_ICC_18_VC141_TEST_SIMD_LAMBDA_RELEASE_BROKEN
(PSTL prefix, of course is not actual)

Reply:
Fixed:
#define _LIBCPP_MSC_TEST_OF_COMPLEX_POW__NOT_BROKEN (_LIBCPP_VERSION || _MSC_VER)
#define _LIBCPP_MSC_OP_EQUAL_FOR_COMPLEX_INT_NOT_BROKEN (_LIBCPP_VERSION || _MSC_VER)